python/lsst/meas/astrom/astrometry.py

```diff
@@ -0,0 +1,257 @@
+from __future__ import absolute_import, division, print_function
+
+import math
+
+import numpy
+
+from lsst.afw.image import DistortedTanWcs, ExposureF
+from lsst.afw.table import Point2DKey
+from lsst.afw.cameraGeom import TAN_PIXELS
+import lsst.pex.config as pexConfig
+import lsst.pipe.base as pipeBase
+from .loadAstrometryNetObjects import LoadAstrometryNetObjectsTask
+from .matchOptimisticB import MatchOptimisticBTask
+from .fitTanSipWcs import FitTanSipWcsTask
+
+class AstrometryConfig(pexConfig.Config):
+    refObjLoader = pexConfig.ConfigurableField(
+        target = LoadAstrometryNetObjectsTask,
+        doc = "reference object loader",
+    )
+    matcher = pexConfig.ConfigurableField(
+        target = MatchOptimisticBTask,
+        doc = "reference object/source matcher",
+    )
+    wcsFitter = pexConfig.ConfigurableField(
+        target = FitTanSipWcsTask,
+        doc = "WCS fitter",
+    )
+    forceKnownWcs = pexConfig.Field(
+        dtype = bool,
+        doc= "Assume that the input image's WCS is correct, without comparing it to any external reality " +
+            "  (but still match reference objects to sources)",
+        default = False,
+    )
```

Comment:
Since it now means "don't run the fitter", how about changing the name of "forceKnownWcs"?  Maybe "doMatchOnly" or "useInitialWcs"?